test/config.test.js

```diff
@@ -1025,6 +1014,171 @@ describe(`configTest: ${printPath("[test/config.test.js]")}`, function () {
+                }
+            }
+
+            resetAll();
+        }
+
+        {
+            try {
+                STExpress.init({
+                    supertokens: {
+                        connectionURI: "http://localhost:8080",
+                        a: "b",
+                    },
+                    appInfo: {
+                        apiDomain: "api.supertokens.io",
+                        appName: "SuperTokens",
+                        websiteDomain: "supertokens.io",
+                        apiBasePath: "/custom/a",
+                    },
+                    recipeList: [Session.init()],
+                });
+            } catch (err) {
+                if (
+                    err.message !==
+                    'Config Schema Error: supertokens is not allowed to have the additional property "a"'
+                ) {
+                    throw err;
+                }
+            }
+
+            resetAll();
+        }
+
+        {
+            try {
+                STExpress.init({
+                    supertokens: {
+                        connectionURI: "http://localhost:8080",
+                    },
+                    appInfo: {
+                        appName: "SuperTokens",
+                        websiteDomain: "supertokens.io",
+                        apiBasePath: "/custom/a",
+                    },
+                    recipeList: [Session.init()],
+                });
+            } catch (err) {
+                if (err.message !== 'Config Schema Error: appInfo requires property "apiDomain"') {
+                    throw err;
+                }
+            }
+
+            resetAll();
+        }
+
+        {
+            try {
+                STExpress.init({
+                    supertokens: {
+                        connectionURI: "http://localhost:8080",
+                    },
+                    appInfo: {
+                        apiDomain: "api.supertokens.io",
+                        websiteDomain: "supertokens.io",
+                        apiBasePath: "/custom/a",
+                    },
+                    recipeList: [Session.init()],
+                });
+            } catch (err) {
+                if (err.message !== 'Config Schema Error: appInfo requires property "appName"') {
+                    throw err;
+                }
+            }
+
+            resetAll();
+        }
+
+        {
+            try {
+                STExpress.init({
+                    supertokens: {
+                        connectionURI: "http://localhost:8080",
+                    },
+                    appInfo: {
+                        apiDomain: "api.supertokens.io",
+                        appName: "Supertokens",
+                        websiteDomain: "supertokens.io",
+                        apiBasePath: "/custom/a",
+                    },
+                });
+            } catch (err) {
+                if (err.message !== 'Config Schema Error: requires property "recipeList"') {
```

Comment:
Shouldn't it be '<something> requires property "recipeList"'? Suggestions for <something>
- The root object
- init function (I prefer this)